src/tests/suite/daos_epoch.c

```diff
@@ -352,12 +351,13 @@ test_snapshots(void **argp)
 	/* Reopen container, verify number of snapshots */
 	print_message("Container (re)open nsnapshots=%d lsnapshot="DF_X64"\n", (snap_count-1),
 		      snaps[snap_count-1]);
-	MUST(cont_open(arg, co_uuid, DAOS_COO_RW | DAOS_COO_NOSLIP, &coh));
+	uuid_unparse(co_uuid, uuid_str);
```

Comment:
is this uuid_unparse() redundant with the one performed earlier on line 222? Probably not a big deal to leave this here even if it is functionally redundant. i.e., just a question - up to you if you want to remove this.

Reply:
You are right. I can remove it in another PR. Thank you!

Reply:
Just removed this line. Thank you!